library/globals.inc.php

```diff
@@ -1585,42 +1587,79 @@ function gblTimeZones()
             xl("Allow Check In before the appointment's time.")
         ),
+
+        'submit_changes_for_all_appts_at_once' => array(
+            xl('Submit Changes For All Appts At Once'),
+            'bool',                           // data type
+            '1',                              // default
+            xl('Enables to submit changes for all appointments of a recurrence at once.')
+        ),
+
         'disable_pat_trkr' => array(
-            xl('Disable Patient Flow Board'),
+            xl('Flow Board: Disable '),
             'bool',                           // data type
             '0',                              // default
-            xl('Do not display the patient flow board.')
+            xl('Completely remove the ability to display the Patient Flow Board.')
         ),
 
         'ptkr_visit_reason' => array(
-            xl('Show Visit Reason in Patient Flow Board'),
+            xl('Flow Board: Show Visit Reason'),
             'bool',                           // data type
             '0',                              // default = false
             xl('When Checked, Visit Reason Will Show in Patient Flow Board.')
         ),
 
         'ptkr_show_pid' => array(
-            xl('Show Patient ID in Patient Flow Board'),
+            xl('Flow Board: Show Patient ID'),
             'bool',                           // data type
             '1',                              // default = true
             xl('When Checked, Patient ID Will Show in Patient Flow Board.')
         ),
 
         'ptkr_show_encounter' => array(
-            xl('Show Patient Encounter Number in Patient Flow Board'),
+            xl('Flow Board: Show Encounter Number'),
             'bool',                           // data type
             '1',                              // default = true
             xl('When Checked, Patient Encounter Number Will Show in Patient Flow Board.')
         ),
 
+        'ptkr_show_staff' => array(
+            xl('Flow Board: Show Staff Action'),
+            'bool',                           // data type
+            '1',                              // default = true
+            xl('When Checked, Last Staff to Update Board Will Show in Patient Flow Board.')
+        ),
+
+        'ptkr_date_range' => array(
+            xl('Flow Board: Allow Date Range'),
+            'bool',                          // data type
+            '1',                             // default = true
+            xl('This Allows a Date Range to be Selected in Patient Flow Board.')
+        ),
+
+        'ptkr_end_date' => array(
+            xl('Flow Board: Ending Date'),
+            array(
+                'Y1' => xl('One Year Ahead'),
+                'Y2' => xl('Two Years Ahead'),
+                'M6' => xl('Six Months Ahead'),
+                'M3' => xl('Three Months Ahead'),
+                'M1' => xl('One Month Ahead'),
+                'D1' => xl('One Day Ahead'),
```

Comment:
TODO
@ophthal ,
I am planning to add a D0 (ie. just same day) and default to that.